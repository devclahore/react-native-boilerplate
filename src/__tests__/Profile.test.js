/*
  Author: Taimoor Khan
  GitHub: https://github.com/Taimoormk
  Email: taimoor.m.k AT LIVE.COM 
*/

// ########## Import Dependencies Here ##########
import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";

// ########## Import Components Here ##########
import Profile from "../containers/Profile";

test("renders Profile compomenent correctly", () => {
  const renderer = new ShallowRenderer();
  renderer.render(<Profile />);
  const wrapper = renderer.getRenderOutput();
  expect(wrapper).toMatchSnapshot();
});
