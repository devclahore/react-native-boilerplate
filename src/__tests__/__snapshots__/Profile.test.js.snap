// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Profile compomenent correctly 1`] = `
<React.Fragment>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Text>
      Profile
    </Text>
  </View>
</React.Fragment>
`;
